tasks/Module 3/task_2_test.py

```diff
@@ -0,0 +1,48 @@
+"""Task 2 of Module 3."""
+
+import re
+import pytest
+
+
+def my_splitter(to_split, separator=None):
+    """Create personal str.split() implementation with UnitTest."""
```

Comment:
Remember, that docstring should say what your function does. Currently it looks a bit weird, as your function is supposed to create another split function and unit tests for it. I would rather say something like "Split string with provided separator...".

Reply:
docstring updated